### DISCUSSION TITLE
无线中继和临时 WLAN 热点的区别

### DISCUSSION THREAD

**Discussion by purainity** [Mygod/VPNHotspot#524] [Q&A] [answered]
“无线中继”和“临时 WLAN 热点”这两个功能有什么区别？
What is the difference between "wireless relay" and "temporary WLAN hotspot"?
![IMG_20231216_205623](https://github.com/Mygod/VPNHotspot/assets/126877770/96a7b9ab-488b-4608-bf66-f777982624c9)

**Comment by Mygod** [comment, marked as answer]
Repeater can share your Wi-Fi connection on any device, even those without support for this feature for system hotspot. Temporary hotspot should only be used for bypassing tethering restrictions on system hotspot, and forces ephemeral SSID and passwords.
